projectDocs/Design_Implementation.md

```diff
@@ -0,0 +1,54 @@
+# Design (Subject to updates)
+## Model  
+### Database  
+|   Database column   |   expected input   |        Thoughts on implementation        |
+| ------------------- | ------------------ | ---------------------------------------- |
+|          pk         |    auto-generate   |                                          |
+|       subject       |  string from user  | short description to user to add clarity |
+|  Issue comment FK   |  string from user  | long description to user to add clarity  |
+|     contactName     |  string from user  |                                          |
+|    dateSubmitted    |    auto-generate   |                                          |
+|     issueStatus     | Pulled from github |                                          |
+| github issue number | Pulled from github |                                          |
+|  github repository  | pulled from github |           probably don't need            |
+       
+ ### Reference Codes  
+  |  statusCode  |      StatusLabel      |  
+  | ------------ |  -------------------- |  
+  |      10      |  "Request Received"   |  
+  |      20      |  "Problem Confirmed"  |  
+  |      30      |  "Fix Defined"        |  
+  |      40      |  "Fix Implemented"    |  
+  |      50      |  "Fix Confirmed"      |  
+    
+    
+- Separate comment database will have:  (hard to tell if we need this)
+      - pk
+      - issueFK
+      - date commented
+      - Comment
+      
+### Comment database
+| Database Column |     Expected input     | Thoughts on implementation |
+| --------------- | ---------------------- | -------------------------- |
+|        pk       |     auto-generated     |                            |
```

Comment:
"auto generated" may be a confusing description. Here, only `pk` is `AUTOINCREMENT`, which means that this column in this table will be automatically filled with sequential numbers by MySQL.
`issueFK` is a foreign key-- a primary key value in some other table. In the Issue table it is an `AUTOINCREMENT` field, but not here.